server/routes/enrolled.js

```diff
@@ -1,16 +1,26 @@
 
 const router = require('express').Router();
```

Comment:
Oh you're using router, that's great!  I should have used it on mine to learn it better.  Next time I guess.